client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -22,10 +22,17 @@ describe('FeedbackGeneratorService', function() {
   var CorrectnessTestObjectFactory;
   var ErrorTracebackObjectFactory;
   var FeedbackGeneratorService;
+  var PrereqCheckFailureObjectFactory;
+  var TaskObjectFactory;
   var TracebackCoordinatesObjectFactory;
   var TranscriptService;
   var sampleErrorTraceback;
   var timeLimitErrorTraceback;
+  var testTask;
+
+  var PREREQ_CHECK_TYPE_MISSING_STARTER_CODE = 'missingStarterCode';
```

Comment:
Can we reuse the existing constants by importing them (or hardcode them if we want to guard against them changing)?